src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/organization_policy.py

```diff
@@ -71,10 +68,13 @@ def set_scp_attachment(
     def clean_and_remove_policy_attachment(
             organization_mapping,
             path,
+            policy_filename,
             organizations,
             policy_type):
+        _type = 'scp' if policy_type == "SERVICE_CONTROL_POLICY" else 'tagging-policy'
+        policy_name = 'adf-{0}-{1}--{2}'.format(_type, path, policy_filename) if policy_filename != '{}.json'.format(_type) else 'adf-{0}-{1}'.format(_type, path)
```

Comment:
Could you make this line a function that returns the name to use?

Reply:
Yes, done.